autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -20,10 +21,53 @@ const CourseSelectColumn: React.FC = () => {
   const courseCards = useSelector<RootState, CourseCardArray>(
     (state) => state.courseCards,
   );
-
+  const term = useSelector<RootState, string>((state) => state.term);
   const dispatch = useDispatch();
 
-  const term = useSelector<RootState, string>((state) => state.term);
+  const [styleSheetNum, setStyleSheetNum] = React.useState<number>(null);
+  const [ruleNum, setRuleNum] = React.useState<number>(null);
+  React.useLayoutEffect(() => {
+    if (styleSheetNum === null) {
+      for (let i = 0; i < document.styleSheets.length; i++) {
+        try {
+          const styleSheet = document.styleSheets[i] as CSSStyleSheet;
+          for (let j = 0; j < styleSheet.cssRules.length; j++) {
+            if (styleSheet.cssRules[j].cssText.startsWith('.expanded-row')) {
+              setStyleSheetNum(i);
+              setRuleNum(j);
+            }
+          }
+        // eslint-disable-next-line no-empty
+        } catch (e) {}
+      }
+    }
+  }, [styleSheetNum]);
+
+  const expandedRowRef = React.useRef<HTMLDivElement>(null);
+  React.useLayoutEffect(() => {
+    if (expandedRowRef.current?.children[0].clientHeight < 500 - 8) {
+      const styleSheet = document.styleSheets[styleSheetNum] as CSSStyleSheet;
+      if (styleSheet) {
+        styleSheet.deleteRule(ruleNum);
+        styleSheet.insertRule(
+          `.${styles.expandedRow} {
+            overflow-y: visible;
+          }`, ruleNum,
+        );
+      } else throw Error('styleSheet is undefined');
```

Comment:
A couple things about this error:
- How is this better than just silently ignoring the fact that the ref isn't current?
- If it is better, why doesn't the `else` part of this expression throw an error?

Reply:
Also you can just ignore this if we end up using the changes from my branch